### PR TITLE
Feat/vic 837 admin can create and edit team

### PR DESCRIPTION
**Pull request by adnanalicic**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Review by PhilippFr — DISMISSED** [review]

File: src/main/java/de/caritas/cob/agencyservice/api/service/AppointmentService.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import de.caritas.cob.agencyservice.appointmentservice.generated.web.model.AgencyMasterDataSyncRequestDTO;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class AppointmentService {
```
Comment (tkuzynow) [inline src/main/java/de/caritas/cob/agencyservice/api/service/AppointmentService.java:16]: Perhaps it's better to have @ConditionalOnExpression("${feature.appointment.enabled:true}") on the Bean level and have the if's in the calling service, we usually have done it like this before

Comment (tkuzynow) [inline src/main/java/de/caritas/cob/agencyservice/api/service/AppointmentService.java:16]: please add unit tests for this service

Comment (PhilippFr) [reply src/main/java/de/caritas/cob/agencyservice/api/service/AppointmentService.java:16]: @adnanalicic We've had it like @tkuzynow suggested before, any reason you moved the feature toggles into the Service?

Comment (PhilippFr) [reply src/main/java/de/caritas/cob/agencyservice/api/service/AppointmentService.java:16]: Added

**Review by PhilippFr — APPROVED** [review]

**Pull request merged**